plugins/UM3NetworkPrinting/src/Models.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+from typing import Optional
```

Comment:
I believe the Ultimaker standard says you need to put every class in its own file

Reply:
Yup.